assets/src/components/detourListPage.tsx

```diff
@@ -93,7 +93,10 @@ export const DetourListPage = () => {
           <span className="c-detour-list-page__button-text">Add detour</span>
         </Button>
       )}
+      <h2>Active</h2>
       <DetoursTable data={fakeData} />
+      <h2>Draft</h2>
```

Comment:
This change is here in order to make it easier to actually view the changes while reviewing, but I'm planning on removing it before merging (for fear that it'll create unnecessary conflicts with [this work](https://github.com/mbta/skate/pull/2757/files#diff-9c377c74d3f4b789891f6671a6ff311ea53c18bb05476f0f9ea9ddeffb99f71e)